build.rs

```diff
@@ -292,10 +292,10 @@ mod asm {
             nasm.files(asm_file_paths);
             #[cfg(debug_assertions)]
             nasm.flag("-g");
-            #[cfg(all(debug_assertions, not(windows)))]
-            nasm.flag("-Fdwarf");
-            #[cfg(all(debug_assertions, windows))]
-            nasm.flag("-fwin64");
+            nasm.flag(match os {
```

Comment:
This should still be guarded by `#[cfg(debug_assertions)]` because `-F` implies `-g` in some `nasm` versions.

https://www.nasm.us/xdoc/2.14rc5/html/nasmdoc2.html#section-2.1.12

Reply:
Fixed in 47059d1.